straxen/plugins/online_monitor.py

```diff
@@ -175,3 +175,47 @@ def area_width_hist(self, data):
             range=self.config['area_vs_width_bounds'],
             bins=self.config['area_vs_width_nbins'])
         return hist.T
+    
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_nveto_pmts', 
+        type=int,
+        default=120,
+        help='Number of nVeto PMTs'),
+)
+class nVetoPeakMonitor(strax.Plugin):
+    depends_on = 'hitlets_nv', 'events_nv'
+    provides = 'nVeto_peak_monitor'
+    data_kind = 'nVeto_peak_monitor'
+    __version__ = '0.0.1'

```

Comment:
```suggestion
    depends_on = ('hitlets_nv', 'events_nv')
    provides = 'online_monitor_nv'
    data_kind = 'online_monitor_nv'
    __version__ = '0.0.2'

```
Maye in order to be consistent with the NVeto naming conventions we could change this better to `online_monitor_nv`

Reply:
I modified at lines 203-206 in the new version.